org.freedesktop.Sdk.Extension.php74.json

```diff
@@ -0,0 +1,323 @@
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/krakjoe/apcu/archive/v5.1.18.tar.gz",
+					"sha256": "0ffae84b607bf3bfb87a0ab756b7712a93c3de2be20177b94ebf845c3d94f117"
+				},
+				{
+					"type":"file",
+					"path":"apcu.ini"
+				},
+				{
+					"type": "shell",
+					"commands": [
+						"${FLATPAK_DEST}/bin/phpize"
+					]
+				}
+			]
+		},
+		{
+			"name":"xdebug",
+			"buildsystem":"autotools",
+			"config-opts": [
+				"--with-php-config=/usr/lib/sdk/php74/bin/php-config"
+			],
+			"post-install": [
+				"install -Dm755 modules/xdebug.so $(php -r 'echo ini_get(\"extension_dir\");')/xdebug.so",
+				"install -Dm644 xdebug.ini ${PHP_INI_DIR}/conf.d/xdebug.ini",
+				"install -Dm644 LICENSE ${FLATPAK_DEST}/share/licenses/xdebug/LICENSE"
+			],
+			"sources":[
+				{
+					"type":"archive",
+					"url":"https://xdebug.org/files/xdebug-2.8.0.tgz",
+					"sha256":"cb1d117ecbec7409a408cf24e59d92f7e3816fe8e2331dd0ffc3dce60228c264"
+				},
+				{
+					"type":"file",
+					"path":"xdebug.ini"
+				},
+				{
+					"type": "shell",
+					"commands": [
+						"${FLATPAK_DEST}/bin/phpize"
+					]
+				}
+			]
+		},
+		{
+			"name": "composer",
+			"buildsystem": "simple",
+			"build-commands": [
+				"install -Dm755 composer.phar ${FLATPAK_DEST}/bin/composer",
+				"install -Dm644 LICENSE ${FLATPAK_DEST}/share/licenses/composer/LICENSE"
+			],
+			"sources": [
+				{
+					"type": "file",
+					"url": "https://getcomposer.org/download/1.9.1/composer.phar",
+					"sha256": "1f210b9037fcf82670d75892dfc44400f13fe9ada7af9e787f93e50e3b764111"
+				},
+				{
+					"type": "git",
+					"url": "https://github.com/composer/composer.git",
+					"tag": "1.9.1",
+					"commit": "bb01f2180df87ce7992b8331a68904f80439dd2f"
+				}
+			]
+		},
+		{
+			"name": "scripts",
+			"buildsystem": "simple",
+			"build-commands": [
+				"cp {enable,install{,-sdk}}.sh ${FLATPAK_DEST}",
+				"install -Dm0644 ${FLATPAK_ID}.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml",
+				"appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}"
+			],
+			"sources": [
+				{
+					"type": "script",
+					"dest-filename": "install.sh",
+					"commands": [
+						"install -Dm0755 /usr/lib/sdk/php74/bin/php ${FLATPAK_DEST}/php74/bin/php",
+						"install -Dm0755 /usr/lib/sdk/php74/bin/phpdbg ${FLATPAK_DEST}/php74/bin/phpdbg",
+						"install -Dm0755 /usr/lib/sdk/php74/bin/composer ${FLATPAK_DEST}/php74/bin/composer"
+					]
+				},
+				{
+					"type": "script",
+					"dest-filename": "install-sdk.sh",
+					"commands": [
+						"cp -r /usr/lib/sdk/php74 ${FLATPAK_DEST}/php74"
+					]
+				},
+				{
+					"type": "script",
+					"dest-filename": "enable.sh",
+					"commands": [
+						"export PATH=$PATH:/usr/lib/sdk/php74/bin:/var/config/composer/vendor/bin"
+					]
+				},
+				{
+					"type": "file",
+					"path": "org.freedesktop.Sdk.Extension.php74.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Done.